api/src/pcapi/core/search/commands/indexation.py

```diff
@@ -233,10 +233,8 @@ def _reindex_all_venues(algolia_batch_size: int, max_venues: int) -> None:
 
 
 @blueprint.cli.command("reindex_offers_if_ean_booked_recently")
-@click.option("--since", help="Number of hours since last booking to reindex", type=int, default=24)
-@click.option("--batch-size", help="Batch size (Algolia)", type=int, default=10_000)
-@click.option("--max-offers", help="Max number of offers (total)", type=int, default=10_000)
-def reindex_offers_if_ean_booked_recently(since: int, batch_size: int, max_offers: int) -> None:
+@click.option("--since", help="the number of days to consider in the booking count", type=int, default=30)
+def reindex_offers_if_ean_booked_recently(since: int) -> None:
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
**💣 chore:** ‏Renommer